tidy3d/components/eme/simulation.py

```diff
@@ -175,11 +176,13 @@ class EMESimulation(AbstractYeeGridSimulation):
         validate_default=True,
     )
 
-    subpixel: bool = pd.Field(
-        True,
+    subpixel: Union[bool, SubpixelSpec] = pd.Field(
+        SubpixelSpec(),
         title="Subpixel Averaging",
         description="If ``True``, uses subpixel averaging of the permittivity "
-        "based on structure definition, resulting in much higher accuracy for a given grid size.",
+        "based on structure definition, resulting in much higher accuracy for a given grid size. "
```

Comment:
I think we need to flip this message around and say what SubpixelSpec does; then say what setting `subpixel=True` corresponds to in terms of a SubpixelSpec (dielectric subpixel, everything else staircased).

Reply:
when `subpixel=True`, now conformal is also applied to PEC, as it is in general much more accurate than staircasing.